components/http_foundation/sessions.rst

```diff
@@ -190,8 +190,12 @@ data is an array, for example a set of tokens. In this case, managing the array
 becomes a burden because you have to retrieve the array then process it and
 store it again::
 
-    $tokens = array('tokens' => array('a' => 'a6c1e0b6',
-                                      'b' => 'f4a7b1f3'));
+    $tokens = array(
+        'tokens' => array(
+            'a' => 'a6c1e0b6',
+            'b' => 'f4a7b1f3',
+        )
```

Comment:
Yes sorry, missed that comma, will fix soon.

At first I also did as @xabbuh suggested, but decided against it as it doesn't make it as clear.